Formula/plugin-modernizer.rb

```diff
@@ -0,0 +1,17 @@
+class PluginModernizer < Formula
+    desc "Plugin Modernizer"
+    version "911.v211a_1d5a_7a_a_f".split(".")[0]
```

Comment:
```suggestion
    # Note: Brew don't really like our versions scheme for CD. Implicitly it consider "499.vb_86f97f0b_197" 
    # as version "197" which is incorrect. So using version which only "%d" part for CD
    version "911.v211a_1d5a_7a_a_f".split(".")[0]
```

nitpicking: your note is really useful, It could be nice to comment the source as well